### PR TITLE
Adding CSS Handles in orderOptions Component

### PR DESCRIPTION
**Pull request by Caroladao**
#### What does this PR do? \*
Adding CSS Handles in orderOptions Component

#### How to test it? \*
[Workspace](https://task238--menudigitalmx.myvtex.com/checkout/orderPlaced/?og=1067610540886)

image

### CODE REVIEW

**Comment by Caroladao** [comment]
@klzns you can see this PR?

**Review by klzns — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -313,6 +313,11 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | `customerInfoListEmail`   |
 | `customerInfoListDocument`   |
 | `customerInfoListPhone`   |
+| `printReceiptButton`   |
+| `updateOrderButton`   |
+| `myOrdersButton`   |
+| `takeAwayCancelButton`   |
+| `cancelButton`   |
```
Comment (kaisermann) [inline docs/README.md]: Can you order this alphabetically?

File: react/OrderOptions.tsx [hunk 2]
```diff
@@ -28,56 +28,78 @@ interface Props {
   orderId?: string
 }
 
+const CSS_HANDLES = [
+  'printReceiptButton',
+  'updateOrderButton',
+  'myOrdersButton',
+  'takeAwayCancelButton',
+  'cancelButton'
+]
```
Comment (kaisermann) [inline react/OrderOptions.tsx]:
```suggestion
] as const
```

**Review by kaisermann — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (Caroladao) [reply docs/README.md]: yes

**Comment by Caroladao** [comment]
@kaisermann it's right?

**Comment by Caroladao** [comment]
@kaisermann I have already done the changes, could you validate please?

**Pull request closed without merging**